packages/editor/src/components/EditorContainer.tsx

```diff
@@ -168,12 +169,19 @@ const EditorContainer = () => {
     }
   }, [scenePath.value])
 
-  const viewerEntity = useMutableState(EngineState).viewerEntity.value
+  useEffect(() => {
+    initializeSpatialEngine()
+    return () => {
+      destroySpatialEngine()
+    }
+  }, [])
```

Comment:
I'd like to better understand how often this `useEffect` is run; it has no dependencies, but it initializes and destroys some things that look like a big deal, so to my undertrained eye it looks conspicuous

Reply:
"Runs only once, when the editor container is mounted."